CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/), and this
 project adheres to [Semantic Versioning](https://semver.org/).
 
+## [0.6.1] - 2023-02-21
```

Comment:
Mind changing the date to TBD? I'm going to have to merge this and then fix macOS CI